### PR TITLE
[Go] Add tango

### LINKED ISSUE #2506: Add tango
Hi @lunny ,

Would you mind to add (or consent that we add) [tango](https://gitea.com/lunny/tango
) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

### PR DESCRIPTION
**Pull request by waghanza**
Hi @lunny,

This `PR` add `tango`. 

Could you please check this code, and give me your advices :heart: 

Purpose is to have **production-grade** implementations `here` so we can compare (performances) on factual basis.

Regards,

------------------------------------

Closes #2506

### CODE REVIEW

File: go/tango/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+func (Empty) Get() string {
+	return ""
+}
+
+func (Empty) Post() string {
+	return ""
+}
+
+func main() {
+	t := tango.Classic()
```
Comment (lunny) [inline go/tango/main.go:26]: I suggest to use 
```
t := tango.New(
		tango.Logging(),
		tango.Recovery(false),
		tango.Return(),
		tango.Param(),
	)
```

And the default port is 8000.

**Comment by lunny** [comment]
And please mark as `language:golang` but not `language:php`.

**Comment by waghanza** [comment]
> And please mark as language:golang but not language:php.

Oops

File: go/tango/main.go (hunk 1 above)
Comment (waghanza) [reply go/tango/main.go:26]: sure :tada: 

However, the port is set by an environment variable

**Pull request merged**